src/python/CRABClient/Commands/status.py

```diff
@@ -908,7 +908,7 @@ def printPublication(self, publicationEnabled, jobsPerStatus, numProbes, numUnpu
             self.logger.info(msg)
             return pubStatus
         ## Case in which there was an error in retrieving the publication status.
-        pubSource = "\n(from Oracle transfer DB)"
+        pubSource = "\n(from internal CRAB transfersDB)"
```

Comment:
sorry, names matter. The text I propose is better.
"CRAB internal bookkeeping" is something which users can understand
transferdb (no uppercase) is the name of the table inside CRAB ORACLE DB which is used to keep track, i.e. it is the details needed for experts.
There is no "transfer DataBase " in CRAB separate from other DataBase(s). The table name is not ideal, but that was not something I picked and fear it is too much work to put a decent name there.